pages/collection.html

```diff
@@ -78,8 +78,22 @@ <h1>The Regis Altare Birthday Project 2023</h1>
       <section>
         <button id="full-collection-toggle">Show Full Collection</button>
         <button id="reset-collection">Reset Card Collection</button>
+        <label>
+            Sort by:
```

Comment:
I think this addition is causing a white border around the collection area and the rounded corners that were there before got lost. Could that be addressed in this PR or in a quick followup?

Reply:
The white border is caused by the section element that begins on line 78 and ends on line 98. I marked it with the "collection-border" id. It wraps the "card-list" section which is the one on blue and it's needed for the controls as everything in that section is replaced on card render. I did notice that deleting the "collection-border" doesn't bring back the round corners, so something else must be happening. I'll investigate it.

Reply:
Actually I think the round borders is no longer there after Bon finalized the "tabs" page styling - now it's sharp borders with colored backgrounds. So it's an issue that the background is white, but the corners don't need to be round. I've added a ticket to address this on our Notion board